src/engine/strat_engine/pool.rs

```diff
@@ -129,6 +129,12 @@ impl StratPool {
     pub fn has_filesystems(&self) -> bool {
         self.thin_pool.has_filesystems()
     }
+
+    pub fn snapshot_filesystem(&mut self, filesystem_uuid: Uuid) -> EngineResult<FilesystemUuid> {
+        let dm = DM::new()?;
+        Ok(self.thin_pool
```

Comment:
I think you can just leave off the Ok...? part, i.e., instead of ```Ok(x?)``` just do ```x```.

Reply:
ok